components/gitpod-db/src/long-running-migration/workspace-organizationid-migration.spec.db.ts

```diff
@@ -52,7 +52,7 @@ describe("Workspace organizationid migration", () => {
             },
         });
 
-        const nowMinus30 = new Date(now.getTime() - 24 * 60 * 60 * 1000 * 30);
+        const nowMinus30 = new Date(now.getTime() - 24 * 60 * 60 * 1000);
```

Comment:
```suggestion
        const nowMinus1000 = new Date(now.getTime() - 24 * 60 * 60 * 1000);
```
? perhaps rename to "oldWorksapceTimestamp"